sceptre/cli/diff.py

```diff
@@ -112,9 +114,9 @@ def diff_command(
     if differ == "deepdiff":
         stack_differ = DeepDiffStackDiffer(show_no_echo)
         writer_class = DeepDiffWriter
-    elif differ == 'difflib':
+    elif differ == "difflib":
```

Comment:
lol. So you're one of _those_ developers? 😆

Reply:
I worked with a guy once who was really OCD about quote consistency. So I would intentionally alternate between quote-types line-by-line just to mess with him.

Reply:
Is he ok?

Reply:
Anyway, here's the Truth.
https://anti-pattern.com/always-use-double-quoted-strings-in-ruby

Reply:
Ha ha ha ha ha.